charts/vela-core/templates/defwithtemplate/manualscale.yaml

```diff
@@ -18,7 +18,8 @@ spec:
         	spec: replicas: parameter.replicas
         }
         parameter: {
-        	//+short=r
+        	// +usage=specify the number of workload
+        	// +short=r
```

Comment:
`+short` can be deleted as cli trait operation is not supported any more.

Reply:
got it